### PR TITLE
Add Visible -> Visibility to migration guide

### PR DESCRIPTION
**Pull request by ramirezmike**
This adds a note on the changes with Visibility. I'm open to suggestions on the is_transparent text

### CODE REVIEW

**Review by IceSentry — APPROVED** [review]

File: content/learn/book/migration-guides/0.5-0.6/_index.md [hunk 1]
```diff
@@ -272,4 +272,38 @@ let xyz: IVec3 = xyz.as_ivec3();
 ### StandardMaterial's "roughness" is renamed to "perceptual_roughness"
 The {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="StandardMaterial" no_mod=true)}} field `roughness` was renamed to `perceptual_roughness`.
 
+### Visible is now Visibility 
+The {{rust_type(type="struct" crate="bevy" mod="render::draw" version="0.5.0" name="Visible" no_mod=true)}} struct, which is used in a number of components to set visibility, was renamed to {{rust_type(type="struct" crate="bevy" mod="render::view" version="0.6.0" name="Visibility" no_mod=true)}}. Additionally, the field `is_transparent` was removed from the struct. For 3D, transparency can be set using the `alpha_mode` field on a material. In 2D, everything is transparent.
```
Comment (alice-i-cecile) [inline content/learn/book/migration-guides/0.5-0.6/_index.md:276]:
```suggestion
The {{rust_type(type="struct" crate="bevy" mod="render::draw" version="0.5.0" name="Visible" no_mod=true)}} struct, which is used in a number of components to set visibility, was renamed to {{rust_type(type="struct" crate="bevy" mod="render::view" version="0.6.0" name="Visibility" no_mod=true)}}. Additionally, the field `is_transparent` was removed from the struct. For 3D, transparency can be set using the `alpha_mode` field on a material. Transparency is automatically enabled for all objects in 2D. 
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Small wording fix, then this LGTM.

File: content/learn/book/migration-guides/0.5-0.6/_index.md [hunk 2]
```diff
@@ -273,6 +273,37 @@ let xyz: IVec3 = xyz.as_ivec3();
...
 ### Visible is now Visibility 
-The {{rust_type(type="struct" crate="bevy" mod="render::draw" version="0.5.0" name="Visible" no_mod=true)}} struct, which is used in a number of components to set visibility, was renamed to {{rust_type(type="struct" crate="bevy" mod="render::view" version="0.6.0" name="Visibility" no_mod=true)}}
+The {{rust_type(type="struct" crate="bevy" mod="render::draw" version="0.5.0" name="Visible" no_mod=true)}} struct, which is used in a number of components to set visibility, was renamed to {{rust_type(type="struct" crate="bevy" mod="render::view" version="0.6.0" name="Visibility" no_mod=true)}}. Additionally, the field `is_transparent` was removed from the struct. For 3D, transparency can be set using the `alpha_mode` field on a material. In 2D, everything is transparent. 
 
-Additionally, the field `is_transparent` was removed from the struct. For 3D, transparency can be set using the `alpha_mode` field on a material. In 2D, everything is transparent. 
+
+```rust
+// 0.5
+let material_handle = materials.add(StandardMaterial {
+                          base_color_texture: Some(texture.clone()),
```
Comment (rparrett) [inline content/learn/book/migration-guides/0.5-0.6/_index.md:282]: What's going on with the indentation here?

Comment (KDecay) [reply content/learn/book/migration-guides/0.5-0.6/_index.md:282]: Looks good, except for the indentation like @rparrett already mentioned.
I would change the indentation to the following:

```rust
    let material_handle = materials.add(StandardMaterial {
        base_color_texture: Some(texture.clone()),
        ..Default::default()
    });
```

```rust
    let material_handle = materials.add(StandardMaterial {
        base_color_texture: Some(texture.clone()),
        alpha_mode: AlphaMode::Blend,
        ..Default::default()
    });
```

**Review by KDecay — CHANGES_REQUESTED** [review]

**Comment by alice-i-cecile** [comment]
@ramirezmike can you rebase to fix conflicts, then clean up the indentation? Once that's done this looks good to merge.

**Pull request merged**